.github/workflows/doxygen-gh-pages.yml

```diff
@@ -0,0 +1,39 @@
+name: Deploy Doxygen results to Github Pages
+
+on: push
+#on:
+#  push:
+#    branches:
+#      - main
+
+permissions:
+  contents: write
+jobs:
+  build-and-deploy:
+    concurrency: ci-${{ github.ref }} # Recommended if you intend to make multiple deployments in quick succession.
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v3
+      - name: Prepare dependencies
+        run: sudo apt-get update && sudo apt-get -yq install libasio-dev doxygen mkdocs graphviz zlib1g-dev gcc clang make cmake python3 python3-pip git openssl libssl-dev
+      - name: prepate pip dependencies
+        run: pip3 install mkdocs-material mkdocs-redirects pyyaml mkdocs-meta-descriptions-plugin --no-input
+      - name: configure
+        run: cmake -B build -S . -DCROW_BUILD_DOCS=ON
```

Comment:
I'm a little confused as to the purpose of this command. As far as I know `CROW_BUILD_DOCS` isn't defined anywhere in `CMakeLists.txt`

Reply:
yes, not yet used. I was thinking about running doxygen out of cmake build, but this is not realized.

Reply:
okay, could we keep it commented out until it's ready, so that it doesn't add unnecessary work.